crates/frontend/src/components/experiment_form/utils.rs

```diff
@@ -66,14 +75,19 @@ pub async fn update_experiment(
     org_id: String,
     description: String,
     change_reason: String,
+    experiment_group_id: Option<String>,
 ) -> Result<ExperimentResponse, String> {
+    let experiment_group_id = experiment_group_id
+        .map(|id| id.parse::<i64>())
+        .transpose()
+        .map_err(|_| "Invalid experiment group ID".to_string())?;
     let payload = OverrideKeysUpdateRequest {
         variants: Result::<Vec<VariantUpdateRequest>, String>::from_iter(variants)?,
         metrics,
         description: Some(Description::try_from(description)?),
         change_reason: ChangeReason::try_from(change_reason)?,
+        experiment_group_id: experiment_group_id.map(I64Update::Add).map(I64Update::from),
```

Comment:
this is wrong, over here one cannot delete the data
rather do how it was done for config_versions, pass the value of experiment_group_id as `serde_json::Value`
`Value::String(id)` when the data is present and `Value::Null` when the data is to be removed

and over here write this:
```suggestion
        experiment_group_id: Some(
            serde_json::from_value(experiment_group_id)
                .map_err(|e| format!("Invalid experiment group id: {e}"))?,
        ),
```

and changing the function parameter:
```
        experiment_group_id: Value,
```

also, this same above suggested changes would be needed to be done in the create function as well